osu.Game/Screens/OnlinePlay/Lounge/DrawableLoungeRoom.cs

```diff
@@ -155,7 +158,7 @@ public bool MatchingFilter
 
         public Popover GetPopover() => new PasswordEntryPopover(Room);
 
-        public MenuItem[] ContextMenuItems
+        public new MenuItem[] ContextMenuItems
```

Comment:
I'm not sure how this works with `new`, but you should probably use `virtual` instead.

And once you use that, you can call `base.ContextMenuItems` rather than duplicating the code here.